internal/subscription/global_limits.go

```diff
@@ -23,6 +23,10 @@ const (
 	MaxDeploymentTargetsPerCustomerOrganizationStarter   Limit = 1
 	MaxDeploymentTargetsPerCustomerOrganizationPro       Limit = 3
 	MaxDeploymentTargetsPerCustomerOrganizationUnlimited Limit = Unlimited
+
+	MaxLogExportRowsStarter    Limit = 100
+	MaxLogExportRowsPro        Limit = 10_000
+	MaxLogExportRowsEnterprise Limit = 1_000_000
```

Comment:
we don't keep more than 5000

Reply:
good point we should increase it to 10_000 via the env

Reply:
and for future enterprise / dedicated installations we can come up with a solution later on